python/taichi/ui/scene.py

```diff
@@ -106,7 +116,39 @@ def lines(self,
                 If `per_vertex_color` is provided, this is ignored.
             per_vertex_color (Tuple[float]): a taichi 3D vector field, where each
                 element indicate the RGB color of the line.
+            vertex_offset: int type(otherwise float type will be floored to int),
```

Comment:
https://github.com/taichi-dev/taichi/blob/master/python/taichi/ui/gui.py#L20-L30 please follow this format for args
e.g. 
```
vertex_offset (int) : ...

Reply:
Okay, I will follow that.

Reply:
Already done.